pkgs/by-name/an/ansi-cheatsheet/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  fetchFromGitHub,
+  lib,
+  stdenv,
+  zig,
+}:
+stdenv.mkDerivation {
+  pname = "ansi";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "NewDawn0";
+    repo = "ansi";
+    rev = "6b6d3b991706247af90f3f72c67238144f77a928";
```

Comment:
Why not use a named tag?

Reply:
I'm Woking on it and will submtit a cmmit with all the applied changes soon